src/math/curve.js

```diff
@@ -39,8 +51,8 @@ Object.assign(pc, (function () {
     var Curve = function (data) {
         this.keys = [];
         this.type = CURVE_SMOOTHSTEP;
-
         this.tension = 0.5; // used for CURVE_CARDINAL
+        this.it = new pc.CurveIterator(this);
```

Comment:
Should "it" be a private variable? If so prefix it with _ otherwise we need to document it as a public property.

Reply:
renamed to _eval